src/ui/properties/ImageNodeEditor.js

```diff
@@ -38,6 +40,9 @@ export default function ImageNodeEditor(props) {
       <InputGroup name="Billboard" info="Image always faces user in Hubs.">
         <BooleanInput value={node.billboard} onChange={onChangeBillboard} />
       </InputGroup>
+      <InputGroup name="Link Href" info="Allows the image to function as a link for the given url.">
```

Comment:
Should we hide this if projection is not set to flat since we don't export in that case?